pytensor/tensor/rewriting/linalg.py

```diff
@@ -966,3 +970,30 @@ def rewrite_cholesky_diag_to_sqrt_diag(fgraph, node):
             non_eye_input = pt.shape_padaxis(non_eye_input, -2)
 
     return [eye_input * (non_eye_input**0.5)]
+
+
+@node_rewriter([Blockwise])
```

Comment:
You can track the instantiated Blockwise Op directly

Reply:
In that case I can remove the `core_op` check?

Reply:
Also I'm doing `Blockwise` inside dispatching functions, how do I handle that case? I don't have an instantiated blockwise Op to refer to